PED/SetRelationshipGroupDontAffectWantedLevel.md

```diff
@@ -0,0 +1,18 @@
+---
+ns: NETWORK
+aliases: ["0x5615E0C5EB2BC6E2"]
+---
+## _SET_RELATIONSHIP_GROUP_DONT_AFFECT_WANTED_LEVEL
+
+```c
+// 0x5615E0C5EB2BC6E2
+void _SET_RELATIONSHIP_GROUP_DONT_AFFECT_WANTED_LEVEL(Hash group, BOOL p1);
```

Comment:
Changed `Any` type to something else which would break ABI compat _if_ anyone uses this native already. Since it's likely not the case, ~fine.